.github/actions/nm-build-vllm/action.yml

```diff
@@ -24,6 +24,7 @@ runs:
     run: |
       COMMIT=${{ github.sha }}
       VENV="${{ inputs.venv }}-${COMMIT:0:7}"
+
```

Comment:
why the extra whitespace?

Reply:
Leftover from when the injection code was here; removed.